department-of-veterans-affairs/va.gov-cms
# Removed flaky json api explorer unit test

## Description
Closes #18199

## Testing done
Testing will be done on tugboat

## Screenshots


## QA steps

What needs to be checked to prove this works?

- Verify that there are only 301 PHP unit tests run in the tugboat logs.

What needs to be checked to prove it didn't break any related things?

- If the automated tests pass, this is working correctly.


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
